.github/CODEOWNERS

```diff
@@ -71,22 +105,52 @@
 /docs/framework/configure-apps/file-schema/wcf/** @HongGit
 /docs/framework/data/wcf/** @HongGit
 /docs/framework/wcf/** @HongGit
-# WPF
-/docs/framework/wpf/** @Thraka
-# Winforms
-/docs/framework/winforms/** @Thraka
 
 ################## .NET STANDARD ##################
-# Analyzers:
-/docs/standard/analyzers/** @BillWagner
+# Analyzers
+/docs/standard/analyzers/** @mairaw
+# Assembly
+/docs/standard/assembly/** @gewarren
+# Asynchronous Programming Patterns
+/docs/standard/asynchronous-programming-patterns/** @gewarren
```

Comment:
I've got responsibility for async in C# (along with threading). This is related. Also, happy to work with @gewarren to make sure these areas are seamless.

Reply:
I think this makes sense @BillWagner. We should ensure that no one is overwhelmed by the number of PRs.